keyboards/wookong/wk87/info.json

```diff
@@ -0,0 +1,266 @@
+            {"matrix": [2, 11], "x": 150, "y": 28, "flags": 4},
+            {"matrix": [2, 12], "x": 163, "y": 28, "flags": 4},
+            {"matrix": [2, 13], "x": 179, "y": 28, "flags": 4},
+            {"matrix": [2, 14], "x": 198, "y": 28, "flags": 1},
+            {"matrix": [2, 15], "x": 211, "y": 28, "flags": 1},
+            {"matrix": [2, 16], "x": 224, "y": 28, "flags": 1},
+            {"matrix": [3, 12], "x": 5, "y": 40, "flags": 8},
+            {"matrix": [3, 11], "x": 23, "y": 40, "flags": 4},
+            {"matrix": [3, 10], "x": 36, "y": 40, "flags": 4},
+            {"matrix": [3, 9], "x": 49, "y": 40, "flags": 4},
+            {"matrix": [3, 8], "x": 62, "y": 40, "flags": 4},
+            {"matrix": [3, 7], "x": 75, "y": 40, "flags": 4},
+            {"matrix": [3, 6], "x": 88, "y": 40, "flags": 4},
+            {"matrix": [3, 5], "x": 101, "y": 40, "flags": 4},
+            {"matrix": [3, 4], "x": 114, "y": 40, "flags": 4},
+            {"matrix": [3, 3], "x": 127, "y": 40, "flags": 4},
+            {"matrix": [3, 2], "x": 140, "y": 40, "flags": 4},
+            {"matrix": [3, 1], "x": 153, "y": 40, "flags": 4},
+            {"matrix": [3, 0], "x": 174, "y": 40, "flags": 1},
+            {"matrix": [4, 0], "x": 8, "y": 52, "flags": 1},
+            {"matrix": [4, 1], "x": 29, "y": 52, "flags": 4},
+            {"matrix": [4, 2], "x": 42, "y": 52, "flags": 4},
+            {"matrix": [4, 3], "x": 55, "y": 52, "flags": 4},
+            {"matrix": [4, 4], "x": 68, "y": 52, "flags": 4},
+            {"matrix": [4, 5], "x": 81, "y": 52, "flags": 4},
+            {"matrix": [4, 6], "x": 94, "y": 52, "flags": 4},
+            {"matrix": [4, 7], "x": 107, "y": 52, "flags": 4},
+            {"matrix": [4, 8], "x": 120, "y": 52, "flags": 4},
+            {"matrix": [4, 9], "x": 133, "y": 52, "flags": 4},
+            {"matrix": [4, 10], "x": 146, "y": 52, "flags": 4},
+            {"matrix": [4, 11], "x": 171, "y": 52, "flags": 1},
+            {"matrix": [4, 15], "x": 211, "y": 52, "flags": 1},
+            {"matrix": [5, 16], "x": 2, "y": 64, "flags": 1},
+            {"matrix": [5, 15], "x": 18, "y": 64, "flags": 1},
+            {"matrix": [5, 14], "x": 34, "y": 64, "flags": 1},
+            {"matrix": [5, 11], "x": 83, "y": 64, "flags": 4},
+            {"matrix": [5, 10], "x": 131, "y": 64, "flags": 1},
+            {"matrix": [5, 9], "x": 148, "y": 64, "flags": 1},
+            {"matrix": [5, 8], "x": 164, "y": 64, "flags": 1},
+            {"matrix": [5, 5], "x": 180, "y": 64, "flags": 1},
+            {"matrix": [5, 2], "x": 198, "y": 64, "flags": 1},
+            {"matrix": [5, 1], "x": 211, "y": 64, "flags": 1},
+            {"matrix": [5, 0], "x": 224, "y": 64, "flags": 1}
+        ]
+    },
+    "url": "https://github.com/hypharf",
+    "usb": {
+        "device_version": "1.0.0",
+        "force_nkro": true,
+        "pid": "0x6322",
+        "vid": "0x314B"
```

Comment:
This vendor ID is in use, and this would need to be changed. 
https://www.the-sz.com/products/usbid/index.php?v=0x314B&p=&n=

Anything past 0x4000 should be okay, though the further past it, the better, as the 0x3xxx block is being actively assigned and is pretty far along. 

Also, pid.codes and the like can be petitioned for a PID for "official" usage.

Reply:
Hi @drashna , our keyboard uses Geehy Semiconductor's APM32 and we have agreement of using their VID 0x314B, if in this way we still need to change vendor ID past 0x4000?